pages/Documents/ConversationalAI/Third-PartyBots/public-api.md

```diff
@@ -39,21 +39,30 @@ To identify the Third-Party Bots API endpoint user, first get domain information
 | z2.bot-connectors.liveperson.net | https://bot-platform-api.emea.fs.liveperson.com/ |
 | z3.bot-connectors.liveperson.net | https://bot-platform-api.apac.fs.liveperson.com/ |
 
-#### Step 2. Get Bearer Token
+#### Step 2. Create a Public API Bot User
 
-To use our Public API you must perform a login request to Third-Party Bots API domain which will in response send you a Third-Party Bots bearer token. There are two ways in which you can perform the login.
+In order to use Public API you must create a dedicated Public API bot agent/user via LivePerson User Management UI. You must not use this dedicated Public API agent/user user in any other LivePerson business process. For example, Using the same Public API Bot Agent to create a Third-Party Bot or using this Bot Agent user to call other LivePerson APIs. The reason for this requirement is to ensure Public API bot agent's session is not invalidated by any other service/business process.
```

Comment:
For example, using the same Public API bot agent

Reply:
using this bot agent

Reply:
The reason for this requirement is to ensure that the session created by the Public API is not